README.md

```diff
@@ -6,6 +6,7 @@ The Azure Key Vault agent container does the following -
 * It runs before any other container as an init-container
 * It connects to Azure Key Vault using the cluster's service principle
 * It then grabs the desired secrets and/or certificates from Azure Key Vault and stores them in a shared volume (memory only - tmpfs)
+* If a secret refers to a key that is backing a certificate, both private key and certificates are exported as pem
```

Comment:
typo: should be `certificate` (without the 's')

Reply:
fixed